devicon.json

```diff
@@ -4753,6 +4753,24 @@
         "color": "#3d3b47",
         "aliases": []
     },
+    {
+        "name": "norg",
```

Comment:
Please add the altnames property as explained [here](https://github.com/devicons/devicon/wiki/Updating-the-%60devicon.json%60) and shown [here](https://github.com/devicons/devicon/wiki/Example-of-Submitting-Icons-to-the-Repository)

Reply:
Added it, thanks for reminding :)